### PR TITLE
Stop updating bundler in CI

### PR DESCRIPTION
**Pull request by tas50**
We don't update it in our builds anymore. We just use the version that's built into Ruby. This omnibus override was only used for CI and we had to match the versions, which was a real pain and pretty pointless.

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: .expeditor/scripts/bk_linux_exec.sh [hunk 1]
```diff
@@ -31,8 +31,10 @@ echo "--- Config information"
 
 echo "!!!! RUBY VERSION !!!!"
 ruby --version
-echo "!!!! BUNDLE LOCATION !!!!"
-which bundle
+echo "!!!! BUNDLER LOCATION !!!!"
+which bundler
+echo "!!!! BUNDLER VERSION !!!!"
+bundler -v
```
Comment (lamont-granquist) [inline .expeditor/scripts/bk_linux_exec.sh:37]: `bundle -v`?

File: omnibus_overrides.rb [hunk 2]
```diff
@@ -4,7 +4,6 @@
 # NOTE: You MUST update omnibus-software when adding new versions of
 # software here: bundle exec rake dependencies:update_omnibus_gemfile_lock
 override :rubygems, version: "3.1.4" # pin to what ships in the ruby version
-override :bundler, version: "2.1.4" # pin to what ships in the ruby version
```
Comment (lamont-granquist) [inline omnibus_overrides.rb:7]: don't we still pull in the bundler omnibus-software definition from the ruby definition which will then install whatever its default version is?

Comment (tas50) [reply omnibus_overrides.rb:7]: We had to nuke all that junk when ruby integrated it. It causes issues when we need to build stuff for old ruby releases.

File: .expeditor/scripts/bk_linux_exec.sh (hunk 1 above)
Comment (tas50) [reply .expeditor/scripts/bk_linux_exec.sh]: it's an alias, but I updated the 3 places I used bundler not bundle

**Review by lamont-granquist — APPROVED** [review]

**Pull request merged**